aws_lambda_builders/workflows/rust_cargo/actions.py

```diff
@@ -68,18 +69,27 @@ def __init__(
         self._flags = flags
         self._architecture = architecture
         self._subprocess_cargo_lambda = subprocess_cargo_lambda
+        self._runtime = runtime
 
     def build_command(self):
         cmd = [self._binaries["cargo"].binary_path, "lambda", "build"]
         if self._mode != BuildMode.DEBUG:
             cmd.append("--release")
-        if self._architecture == ARM64:
+        # Provided.al2 runtime only has GLIB_2.26 and cargo lambda builds with a higher version by default
+        if self._runtime == "provided.al2":
```

Comment:
Have we validated that this is isolated to `al2`?

Reply:
I tested the hello world template with `al2023` and ran into no issues so its only an `al2` issue as far as I can tell